contracts/src/Teleporter/tests/TestMessenger.sol

```diff
@@ -20,7 +21,7 @@ import {ReentrancyGuard} from "@openzeppelin/contracts@4.8.1/security/Reentrancy
  * @dev TestMessenger is test fixture contract that exercises sending and receiving
  * messages cross chain.
  */
-contract TestMessenger is ReentrancyGuard, TeleporterOwnerUpgradeable {
+contract TestMessenger is ReentrancyGuardUpgradeable, TeleporterOwnerUpgradeable {
```

Comment:
We should think of a different name for `TeleporterOwnerUpgradeable`. Right now, we're mixing the meaning of `*Upgradeable` within the same codebase, which is confusing. As an illustration, to be consistent with the names of other contracts, the name should be `TeleporterOwnerUpgradeableUpgradeable`.

Maybe `UpgradeableTeleporter`? Or `UpgradeableTeleporterReceiver`?  Or bake the registry directly into the name, something like `TeleporterRegistryReceiver`.

Reply:
`TeleporterRegistryConsumer` @michaelkaplan13 suggestion also, or can be `TeleporterRegistryApp`

Reply:
created separate issue https://github.com/ava-labs/teleporter/issues/421